course/templates/course/index.html

```diff
@@ -12,85 +12,148 @@ <h1>{{ welcome_text|safe }}</h1>
 </div>
 {% endif %}
 
-<div class="row course-thumbnails">
-    {% site_advert as advert %}
-    {% if advert %}
+{% if my_instances %}
+<div class="frontpage-panel">
+    <h2 class="panel-heading">{% trans "My courses" %}</h2>
+    <div class="row frontpage">
+        {% for instance in my_instances %}
+        {% with instance_url=instance|url %}
         <div class="col-sm-6 col-md-4 col-lg-3">
-            <div class="thumbnail advert-thumbnail">
-                <div class="image" aria-hidden="true">
-                    {% if advert.image %}
-                        <img src="{{ advert.image }}" alt="">
-                    {% else %}
-                        <h1 class="glyphicon glyphicon-comment"></h1>
-                    {% endif %}
+            <div class="frontpage-course card">
+                <a href="{{ instance_url }}">
+                    <div class="course-image" aria-hidden="true">
+                        {% if instance.image %}
+                        <img class="card-img-top" src="{{ instance.image.url }}" alt="" />
+                        {% else %}
+                        <div class="card-img-top">
+                            <i class="glyphicon glyphicon-book"></i>
+                        </div>
+                        {% endif %}
+                    </div>
+                </a>
+                <div class="card-body">
+                    <h3 class="card-title" lang="{{ instance.language }}">
```

Comment:
Btw. this will yield `|en|`, because the language is a list and is encoded as a string like `|en|fi|` and so on.

Reply:
I checked this by logging it in the views.py-file and to me it yielded a string. Also when I look at the code from browser, it seems correct:

![image](https://user-images.githubusercontent.com/43036333/84653157-922fc680-af15-11ea-8e96-7a3bc942d852.png)


If this does not work for some reason, let me know :)

Reply:
Let me look into this. I think it is a simple string for single language courses and encoded for multi lingual courses. Though, I'm not sure how did I get `|en|` there.. Nevertheless, you can apparently force the A+ to think it's a multilingual course. Just write `|en|fi|` in the language box in the coruse edit page (edit course > course | muokkaa kurssia > kurssi).

Reply:
As talked elsewhere, we will leave this as it is now solved, since it is an improvement to earlier as it fixes the cases of single-language courses. This multilanguage-issue can be solved system-wide later on, to ensure that the solution is more re-usable and sustainable.